pom.xml

```diff
@@ -220,7 +218,7 @@
     <dependency>
       <groupId>com.google.code.findbugs</groupId>
       <artifactId>jsr305</artifactId>
-      <version>3.0.1</version>
+      <version>3.0.2</version>
```

Comment:
Can we change to SpotBugs as findbugs is not maintained anymore? see https://gleclaire.github.io/findbugs-maven-plugin/
and https://spotbugs.readthedocs.io/en/latest/maven.html

Reply:
see answer on base-pom

Reply:
please add comment for TODO: https://synjira.atlassian.net/browse/CELDEV-813